src/server/test_utils.cc

```diff
@@ -199,6 +202,28 @@ void BaseFamilyTest::ResetService() {
 
   const TestInfo* const test_info = UnitTest::GetInstance()->current_test_info();
   LOG(INFO) << "Starting " << test_info->name();
+
+  watchdog_fiber_ = pp_->GetNextProactor()->LaunchFiber([this] {
+    if (!watchdog_done_.WaitFor(30s)) {
```

Comment:
I'd increase this a bit, maybe 100-120s?

Reply:
done